site/zh/blog/new-in-devtools-92/index.md

```diff
@@ -37,11 +37,11 @@ Chromium 议题: [1203241](https://crbug.com/1203241)
 
 ## 支持控制台中的 const 重声明 {: #const-redeclaration }
 
-在已有的对[`let` 和 `class` 重声明](/blog/new-in-devtools-80/#redeclarations)的支持之上，控制台现在也支持 const 重声明了。对于使用控制台来试验新 JavaScript 代码的 Web 开发者来说，无法重声明曾是一个常见的困扰。
+控制台现在也支援 const 重声明，在此之前，DevTools 只支援[`let` 和 `class` 的重声明](/blog/new-in-devtools-80/#redeclarations)。无法在控制台中重声明是开发者们常见的困扰，尤其是当您在试验新 JavaScript 代码的时候。
```

Comment:
I feel like "支持“ might be better than “支援”, since "支援" has more of a meaning of "assist", i.e. actively helping someone/something out of a distress, while "支持" is more neutral, more of a meaning of "support". WDYT?